examples/python/human_pose_tracking/main.py

```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
     return str(destination_path)
 
 
+def get_blueprint() -> rrb.BlueprintLike:
+    return rrb.Viewport(
+        rrb.Horizontal(
+            rrb.Vertical(
+                rrb.Spatial2DView(origin="video", name="Segmentation/reprojection"),
+                rrb.Spatial3DView(origin="person", name="3D pose"),
+            ),
+            rrb.Vertical(
+                rrb.Spatial2DView(origin="video/rgb", name="Raw video"),
```

Comment:
nice touch with the raw input! Description box feels almost a bit crammed being so slender. Makes me realize you lied a little bit on the screenshot? Out of the box there's still a selection panel, not seeing it beeing hidden here

Reply:
tuned it a bit.

Reply:
And updated screenshot 👍🏻